homeassistant/components/rfxtrx/__init__.py

```diff
@@ -121,7 +121,7 @@ def _ensure_device(value):
 
 BASE_SCHEMA = vol.Schema(
     {
-        vol.Optional(CONF_DEBUG, default=False): cv.boolean,
+        vol.Optional(CONF_DEBUG, default=False): cv.deprecated,
```

Comment:
The dict that contains the deprecated key should be wrapped with `cv.deprecated`. The value should still be `cv.boolean` otherwise we'd invalidate deprecated configs.

Reply:
Ok, was too fast there. I'll adjust